sdk/servicebus/service-bus/karma.conf.js

```diff
@@ -117,7 +117,13 @@ module.exports = function(config) {
     // start these browsers
     // available browser launchers: https://npmjs.org/browse/keyword/karma-launcher
     // 'ChromeHeadless', 'Chrome', 'Firefox', 'Edge', 'IE'
-    browsers: ["ChromeHeadless"],
+    browsers: ["ChromeHeadlessNoSandbox"],
+    customLaunchers: {
+      ChromeHeadlessNoSandbox: {
+        base: "ChromeHeadless",
+        flags: ["--no-sandbox", "--disable-web-security"]
```

Comment:
Is the `--disable-web-security` flag needed for the CORS issue?

Reply:
Yes

Reply:
Please add a comment regarding the same.